modules/filestore/store.go

```diff
@@ -93,3 +99,7 @@ func initBlock(fileStoreType utils.FileStoreType, connection, endpoint, bucket s
 		return nil, utils.ErrInvalidParams
 	}
 }
+
+func (m *Module) DoesExists(path string) error {
```

Comment:
Put this in operations.go
- check if file storage is enabled
- check if the operation is authorised (check for read op)
- acquire a RLock before the actual operation